assets/javascripts/discourse/initializers/setup-subscriptions.js

```diff
@@ -23,6 +23,36 @@ export default {
           href: `/u/${user.username}/billing/subscriptions`,
           content: "Billing",
         });
+
+        if (user.admin) {
+          api.modifyClassStatic("model:site-setting", {
+            pluginId: "discourse-subscriptions",
+            update(key, value, opts = {}) {
+              if (key === "discourse_subscriptions_pricing_table") {
```

Comment:
`"discourse_subscriptions_pricing_table"` should be a constant

Reply:
by constant you mean an upper case variable name that is declared somewhat at the top of the file like in this case: https://github.com/discourse/discourse/blob/a38830c009ebb780f1fb395d60dfe8f43f69552c/plugins/chat/assets/javascripts/discourse/services/chat-drawer-size.js#L5 ? thanks for your feedback btw! I will go ahead and address it.